README.windows.md

```diff
@@ -45,9 +45,9 @@ or edit `%USERPROFILE%\.gitconfig` and add/edit the lines:
 # Source distribution
 
 ## Supported build platforms
-- Windows 10: supported (32 and 64 bits)
-- Windows 8: supported (32 and 64 bits)
-- Windows 7: supported (32 and 64 bits)
+- Windows 10: supported (32- and 64-bits)
+- Windows 8: supported (32- and 64-bits)
+- Windows 7: supported (32- and 64-bits)
```

Comment:
This is actually correct without the hyphen since it is "32 bits"; you could also write "32-bit" which is an adjective.